README.md

```diff
@@ -138,6 +160,17 @@ To update the realm configuration on the locally running Keycloak instances:
 [location of repo] $ python update_tdr_realm.py local [update policy option: OVERWRITE/SKIP/FAIL]
 ```
 
+### Update TDR Theme Locally
+
+**Note:** The TDR theme sass is used by the TDR Transfer Frontend. When updating the sass for the theme, ensure that any changes are also implemented in the tdr-transfer-frontend repo: https://github.com/nationalarchives/tdr-transfer-frontend/tree/master/npm/css-src/sass
+* This includes any changes to the `.stylelintrc.json`
+
+1. Make necessary changes to the TDR theme (freemarker templates/sass/static resources)
+2. Run following command from the root directory: `[root directory] $ npm run build-theme`
+3. Rebuild the image locally
+4. Run the local docker image
```

Comment:
How long does rebuilding and running the image take? Is it possible to have a development workflow that involves copying updated files to a running image, and does that make development easier?

Reply:
Added an npm script to update the locally running container without the need to restart

Reply:
Having it as an npm script is really nice, thanks!